### ISSUE TITLE
Add 2 automations on Releases Project Board

### ISSUE THREAD

**Issue by TantienHime** [wpmarketingteam/Marcomms-WP-Releases#8]
**Automation**
[Automatically move your cards](https://docs.github.com/articles/configuring-automation-for-project-boards/) to the right place based on the status and activity of your issues and pull requests.

**Comment by TantienHime** [comment]
When @femkreations and I were working through the social media stuff for 5.8, we realized that this project is not automated by default. I've already added the automation to the "Done" column, but I'm using this issue as a demonstration of how to take a card to an issue and also adding the automation in the project itself.

**Comment by TantienHime** [comment]
After some research, the automations for this project have been added as follows:

1. A new issue automatically goes into the To Do column
2. A re-opened issue will go into the In Progress column
3. A Closed issue will automatically be moved to the Done column when it is closed.

I recommend that we do not use the "Close Issue" button on the board, but that we make a comment on the issue and then close the issue with comment, so that we not only know what was done, but WHY. 

Hypothetically, by my clicking the "Close with Comment" button here, the issue should not only close, but should also be moved from the In Progress column to the Done column. I did have to manually move the issue from To Do to In Progress.
